packages/expo-test-runner/package.json

```diff
@@ -30,8 +30,8 @@
     "@babel/core": "^7.23.7",
     "@types/node": "^22.14.0",
     "@types/node-fetch": "^2.5.12",
-    "eslint-config-universe": "^14.2.0",
-    "expo-module-scripts": "^4.1.3",
+    "eslint-config-universe": "^15.0.0",
+    "expo-module-scripts": "^4.1.2",
```

Comment:
Was `expo-module-scripts` downgrade intentional?

Reply:
No, there was a merge conflict and I missed this.